packages/siopv2-oid4vp-op-auth/src/agent/DidAuthSiopOpAuthenticator.ts

```diff
@@ -11,7 +10,36 @@ import {
   IRemoveSiopSessionArgs,
   IRequiredContext,
 } from '../types/IDidAuthSiopOpAuthenticator'
-import { PresentationSignCallback, VerifiedAuthorizationRequest } from '@sphereon/did-auth-siop'
+import { PresentationSignCallback, SupportedVersion, VerifiedAuthorizationRequest } from '@sphereon/did-auth-siop'
+
+import {
+  AddIdentityArgs,
+  CreateConfigArgs,
+  CreateConfigResult,
+  GetSiopRequestArgs,
+  OnContactIdentityCreatedArgs,
+  OnIdentifierCreatedArgs,
+  RetrieveContactArgs,
+  SendResponseArgs,
+  Siopv2AuthorizationRequestData,
+  Siopv2HolderEvent,
+  DidAuthSiopOpAuthenticatorOptions,
+} from '../types/siop-service'
+import { Siopv2Machine } from '../machine/Siopv2Machine'
+import { Siopv2Machine as Siopv2MachineId, Siopv2MachineInstanceOpts } from '../types/machine'
+import { Loggers, LogMethod, W3CVerifiableCredential } from '@sphereon/ssi-types'
+import {
+  ConnectionType,
+  CorrelationIdentifierType,
+  CredentialRole,
+  Identity,
+  IdentityOrigin,
+  NonPersistedIdentity,
+  Party,
+} from '@sphereon/ssi-sdk.data-store'
+import { siopSendAuthorizationResponse, translateCorrelationIdToName } from '../services/Siopv2MachineService'
+
+const logger = Loggers.DEFAULT.options(LOGGER_NAMESPACE, { methods: [LogMethod.CONSOLE, LogMethod.DEBUG_PKG] }).get(LOGGER_NAMESPACE)
```

Comment:
Please do not provide methods when initializing, so an app can set the defaults they want instead of the SDK/lib deciding what is needed.

Reply:
This is copied from oid4vci-holder so this should be addressed here as well:
https://github.com/Sphereon-Opensource/SSI-SDK/blob/d4a4e681b34e1e5acd81b8a8574c843c3eceed70/packages/oid4vci-holder/src/agent/OID4VCIHolder.ts#L94
https://github.com/Sphereon-Opensource/SSI-SDK/blob/d4a4e681b34e1e5acd81b8a8574c843c3eceed70/packages/ssi-types/src/events/index.ts#L149

Reply:
Ping. Point is that we should not keep repeating if it is done somewhere else as well

Reply:
I removed the methods everywhere but in the .test.ts files last week